src/BaseTemplates/WebAPI.vstemplate

```diff
@@ -1,8 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <VSTemplate Version="3.0.0" Type="Project" xmlns="http://schemas.microsoft.com/developer/vstemplate/2005" xmlns:sdk="http://schemas.microsoft.com/developer/vstemplate-sdkextension/2010">
   <TemplateData>
-    <Name>ASP.NET Core 1 Preview Web API</Name>
-    <Description>A project for creating an ASP.NET Core 1 Preview Web API</Description>
+    <Name>ASP.NET Core Preview Web API</Name>
```

Comment:
@joeloff It looks like all the web vstemplates say preview. I'm not sure what you mean here...